services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx

```diff
@@ -315,6 +325,16 @@ export const ReadyServiceLane: React.FC<{
         />
     );
 
+    const reloadButton = (
```

Comment:
i think the button might fit better to the right side of the lane (where "updated just now" appears as well)